### PR TITLE
[Review][Product] Add reviews on product show page

### PR DESCRIPTION
**Pull request by Niiko**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Related tickets |  |
| License | MIT |

After the removal of SyliusWebBundle, i re-add product reviews on product show page

Here is a capture of visual, that's not perfect, especially for the rating which is not inline in form...

review_product

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/views/Product/show.html.twig [hunk 1]
```diff
@@ -37,3 +37,15 @@
...
+{% block javascripts %}
+    {{ parent() }}
+    <script>
+        var createMessage = "{{ 'sylius.resource.create'|trans({'%resource%': 'sylius.ui.product_review'|trans}, 'flashes') }}";
+        var successHeader  = "{{ 'sylius.ui.success'|trans }}";
+        $(document).ready(function() {
+            initializeAjaxForm(createMessage, successHeader);
+        });
+    </script>
+{% endblock %}
```
Comment (tuka217) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Product/show.html.twig]: Missing blank line

File: src/Sylius/Bundle/ShopBundle/Resources/views/Product/show.html.twig [hunk 2]
```diff
@@ -37,3 +37,15 @@
 
 {% include '@SyliusShop/Product/Show/_associations.html.twig' %}
 {% endblock %}
+
+
```
Comment (tuka217) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Product/show.html.twig]: Redundant blank line

File: src/Sylius/Bundle/ShopBundle/Resources/views/ProductReview/_list.html.twig [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+        <h4>Rating: <div class="ui star rating" data-rating="{{ review.rating }}" data-max-rating="5"></div></h4>
+        <p>{{ review.comment }}</p>
+
+        <div class="ui bottom attached label">
+            {% set author = review.author %}
+            {% if author.firstName is not null and author.lastName is not null %}
+                - Author: {{ author.firstName }} {{ author.lastName|first }}.
+            {% else %}
+                - Author: Guest
```
Comment (Zales0123) [inline src/Sylius/Bundle/ShopBundle/Resources/views/ProductReview/_list.html.twig]: I think both ``Author`` and ``Guest`` should be translations keys.

**Comment by Zales0123** [comment]
Great to see reviews on new shop 👍  However, if new feature is added, it should always be added with Behat scenarios. I think that current Behat architecture is pretty stable and clear - you can look on current scenarios and their implementations, as well as on out [Behat guide](http://docs.sylius.org/en/latest/behat/index.html) before implementing them ;) Good luck!

File: src/Sylius/Bundle/ShopBundle/Resources/views/ProductReview/_list.html.twig (hunk 3 above)
Comment (Niiko) [inline src/Sylius/Bundle/ShopBundle/Resources/views/ProductReview/_list.html.twig]: done

**Comment by Niiko** [comment]
Yes, that's true, i was a little bit lazy, will add them soon

**Comment by Zales0123** [comment]
I think it can be closed, as it has been implemented with all other reviews functionality on shop in https://github.com/Sylius/Sylius/pull/6501. Nevertheless, thank you for the contribution!

**Pull request closed without merging**